dts/bindings/input/st,stmpe811.yaml

```diff
@@ -5,7 +5,7 @@ description: STMPE811 I2C touchscreen controller
 
 compatible: "st,stmpe811"
 
-include: i2c-device.yaml
+include: [i2c-device.yaml, touchscreen-common.yaml]
 
 properties:
   int-gpios:
```

Comment:
would it make sense to drop the `screen-width` and `screen-height` properties below and just used the ones from `touchscreen-common.yaml` (`width` and `height`)?

Reply:
We went with renaming our properties from width/height to screen-height/width, and removed screen-height/width from device-specific bindings.